CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Our versioning strategy is as follows:
 * `[sitecore-jss]` `[templates/nextjs-sxa]` Rewrite logic of handling custom error pages. The error pages rewrite page with error(it's saving status code) instead of redirected ([#1469](https://github.com/Sitecore/jss/pull/1469)) ([#1476](https://github.com/Sitecore/jss/pull/1476))
 * `[templates/nextjs]` Remove .babelrc to (re)enable SWC compilation by default ([#1483](https://github.com/Sitecore/jss/pull/1483))
 * `[sitecore-jss]` Handle null items in graphql layout service. ([#1492](https://github.com/Sitecore/jss/pull/1492))
+* `[sitecore-jss]` `[templates/nextjs-personalize]` Unique Identifier Scope ([#1494](https://github.com/Sitecore/jss/pull/1494))
```

Comment:
This doesn't explain the feature all that clearly. I like what you have in the PR description.
```suggestion
* `[sitecore-jss]` `[templates/nextjs-personalize]` Introduced optional personalize scope identifier to isolate embedded personalization data among XM Cloud Environments that are sharing a Personalize tenant ([#1494](https://github.com/Sitecore/jss/pull/1494))
```